llvm/include/llvm/Demangle/ItaniumDemangle.h

```diff
@@ -497,7 +499,7 @@ class PointerType final : public Node {
     } else {
       const auto *objcProto = static_cast<const ObjCProtoName *>(Pointee);
       s += "id<";
-      s += objcProto->Protocol;
+      s += objcProto->getProtocol();
```

Comment:
Is this change necessary?

Reply:
Yes. Protocol is private field accessed in the subclass.

Reply:
`PointerType` is `ObjCProtoName ` friend, so it has access to private members.

Reply:
Good point. I had compilation failure for the `objcProto->Protocol` before. Now after I revert this change compilation still goes fine. Will remove the change,